cancer_ai/validator/model_persister.py

```diff
@@ -0,0 +1,102 @@
+import bittensor as bt
+from datetime import datetime
+from pydantic import BaseModel, HttpUrl
+from sqlalchemy import create_engine, Column, String, DateTime, PrimaryKeyConstraint
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+Base = declarative_base()
+
+class ModelInfo(BaseModel):
```

Comment:
different name, connected to model persistance

Reply:
model_db.py now